pdf/model/colorspace.go

```diff
@@ -509,14 +509,17 @@ func (this *PdfColorspaceDeviceCMYK) ColorToRGB(color PdfColor) (PdfColor, error
 	y := cmyk.Y()
 	k := cmyk.K()
 
-	c = c*(1-k) + k
-	m = m*(1-k) + k
-	y = y*(1-k) + k
-
+	// Peter Williams. I need a cmyk->rgb conversion that gives r!=g or g!=b when c!=m or m!=y
```

Comment:
https://github.com/unidoc/unidoc/pull/105 is a pull request to unidoc/master that contains none of the changes that you have objected to. Please look at that first.